packages/vaadin-grid/test/column-groups.test.js

```diff
@@ -507,5 +507,319 @@ describe('column groups', () => {
       await nextFrame();
       expect(col19._order).to.be.lessThan(col20._order);
     });
+
+    it('should correctly set column order when we have more than 100 columns', async () => {
```

Comment:
Let's change this test to use its own test suite and make fixture smaller using array methods:

```js
  describe('More than 100 columns', () => {
    let grid;

    beforeEach(() => {
      grid = fixtureSync(`
        <vaadin-grid>
          ${Array.from(Array(101).keys())
            .map((col) => {
              return `
                <vaadin-grid-column-group>
                  <vaadin-grid-column id="col${col}" header="Col ${col}"></vaadin-grid-column>
                </vaadin-grid-column-group>
              `;
            })
            .join('')}
        </vaadin-grid>
      `);
    });

    it('should correctly set column order when we have more than 100 columns', async () => {
      let col99 = grid.querySelector('#col99');
      let col100 = grid.querySelector('#col100');
      await nextFrame();
      expect(col99._order).to.be.lessThan(col100._order);
    });
  });
```